### PR TITLE
Video history/ login pages/listing videos

### PR DESCRIPTION
**Pull request by mdarif6**
- Added video listing page with functionality.
- Added video playing page with functionality.
- Added history and watch later page with out functionality.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,13 +3,25 @@ import logo from "./logo.png";
 import HomePage from "./pages/home-page/HomePage";
 import VideoPage from "./pages/video-page/VideoPage";
 import { Route, Routes } from "react-router-dom";
+import WatchLater from "./pages/watch-later-page/WatchLater";
+import LoginPage from "./pages/login-page/LoginPage";
+import SignupPage from "./pages/signup-page/SignupPage";
+import HistoryPage from "./pages/history-page/HistoryPage";
+import PlaylistListing from "./pages/playlist-listing-page/PlaylistListing";
+import CreatePlaylist from "./pages/create-playlist-page/CreatePlaylist";
```
Comment (anandsr21harshit) [inline src/App.js:11]: Create one index file. use Barrel import

File: src/backend/db/categories.js [hunk 2]
```diff
@@ -6,21 +6,39 @@ import { v4 as uuid } from "uuid";
...
+  //   description:
+  //     "Front-end web development, also known as client-side development is the practice of producing HTML, CSS and JavaScript for a website or Web Application",
+  // },
+  // {
+  //   _id: uuid(),
+  //   categoryName: "Backend Development",
+  //   description:
+  //     "Backend Development is also known as server-side development. It is everything that the users don't see and contains behind-the-scenes activities that occur when performing any action on a website.",
+  // },
   {
```
Comment (anandsr21harshit) [inline src/backend/db/categories.js:27]: Remove this

**Review by anandsr21harshit — APPROVED** [review]

**Review by hrishi82 — APPROVED** [review]
You can remove the commented-out code as mentioned by other reviewers.
Rest of the code looks good to me

File: src/App.js (hunk 1 above)
Comment (mdarif6) [reply src/App.js:11]: yes will try in next merge

File: src/backend/db/categories.js (hunk 2 above)
Comment (mdarif6) [reply src/backend/db/categories.js:27]: yes- will removed once complete page working

**Pull request merged**